### PR TITLE
Improve missing german translations

### PR DESCRIPTION
**Pull request by Fabio1988**

### CODE REVIEW

File: public/base-locales/de.json [hunk 1]
```diff
@@ -109,19 +109,19 @@
...
-  "invasions": "invasion",
-  "portals": "Portals",
+  "invasions": "Invasionen",
+  "portals": "Portale",
   "submissionCells": "Zellen übermitteln",
   "spawnpoint": "Spawnpunkt",
   "spawnpoints": "Spawnpunkte",
   "s2CellLevel": "Level {{level}} S2 Zellen",
   "totalCount": "Komplette Anzahl",
   "submissions": "{{}} ",
```
Comment (Fabio1988) [inline public/base-locales/de.json:119]: this one is missing in english!?

**Comment by Fabio1988** [comment]
Is the text "Substitute Pokemon" on empty gym hardcoded text? :)

**Comment by Petap0w** [comment]
> Is the text "Substitute Pokemon" on empty gym hardcoded text? :)

You need to PR directly the central translation repo
https://github.com/bschultz/pogo-translations/blob/d6203691a707bdb6271d8e97358495a03644ab35/static/locales/de.json#L2

**Comment by Fabio1988** [comment]
@Petap0w thx a lot, didn't know of that repo.. https://github.com/bschultz/pogo-translations/pull/15

File: public/base-locales/de.json (hunk 1 above)
Comment (TurtIeSocks) [reply public/base-locales/de.json:119]: No?

https://github.com/WatWowMap/ReactMap/blob/191081fe4aaaa4d8a3a544a9ab0f84a858df5daf/public/base-locales/en.json#L114

**Pull request merged**